sage-ios/SAGE/SAGE/AddAnnouncementView.swift

```diff
@@ -13,18 +13,22 @@ class AddAnnouncementView: FormView {
     var title =  FormFieldItem()
     var school = FormButtonItem()
     var commentField = FormTextItem()
+    var button = SGButton()
     
     //
     // MARK: - Initialization
     //
-    override init(frame: CGRect) {
+    init(frame: CGRect, edit: Bool) {
```

Comment:
you dont need this boolean. instead in configureWithAnnouncement, you can just make the button visible since you know you will need the delete the preload button if your gonna be preloading the page with information